src/nupic/ntypes/SdrMetrics.hpp

```diff
@@ -127,6 +123,10 @@ class _SDR_MetricsHelper {
         NTA_CHECK( dimensions_ == data.dimensions );
         callback( data, 1.0f / std::min( period_, (UInt) ++samples_ ));
     }
+
+    ~_SDR_MetricsHelper() {
```

Comment:
nit, please use This_ format rather. And If you are at it: `SDRMetricsHelper_`

Reply:
I just changed it to `SDR_MetricsHelper_`.